java/integTest/src/test/java/glide/cluster/ClientTests.java

```diff
@@ -0,0 +1,272 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.cluster;
+
+import static glide.TestConfiguration.CLUSTER_PORTS;
+import static glide.api.BaseClient.OK;
+import static glide.api.models.commands.InfoOptions.Section.SERVER;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+import glide.api.RedisClusterClient;
+import glide.api.models.ClusterValue;
+import glide.api.models.commands.InfoOptions;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClusterClientConfiguration;
+import glide.api.models.configuration.RedisCredentials;
+import glide.api.models.exceptions.ClosingException;
+import glide.api.models.exceptions.RequestException;
+import java.lang.module.ModuleDescriptor;
```

Comment:
```suggestion
import java.lang.module.ModuleDescriptor.Version;
```

Reply:
But better to use `semver` - it is more flexible
https://github.com/orgs/Bit-Quill/projects/4/views/2?itemId=55479660&sliceBy%5Bvalue%5D=Java+Milestone+IV&pane=issue